### ISSUE TITLE
[Candidato] Home página logada

### ISSUE THREAD

**Issue by Jacqueline2021** [SouJunior/products#124]

**Comment by Jacqueline2021** [comment]
@samantacds para correção. Obrigada. 
@LuandaClara para criação da tela. Obrigada.

**Comment by Tainawandelli** [comment]
Definição: Como candidato gostaria de acessar minha área logada.
Importante a US seguir o padrão Como, desejo, para que

EX: Como usuário ( bom especificar se nesse caso é o profissional junior, o recrutador ou quem, bom deixar claro qual a persona), desejo acessar a área logada para que eu visualize minhas informações ( pode falar aqui informações sobre o que )

**Comment by Tainawandelli** [comment]
Ao clicar na foto (no canto esquerdo da imagem), o usuário deve ser direcionado para a página de "alterar perfil" para atualização dos seus dados pessoais e profissionais;
Aqui é a foto do perfil do candidato?, acho que ao invés de falar que é do canto esquerdo pode deixar claro que é a foto do perfil do candidato

**Comment by Tainawandelli** [comment]
No botão de configuração https://github.com/SouJunior/products/issues/128 ao lado da foto, o usuário terá acesso a uma lista de funcionalidades; Acho legal colocar um mini print do ícone

**Comment by Tainawandelli** [comment]
No campo "digite o cargo", uma lista com opções deve aparecer. Ex: quando ele digitar "analista", a seguinte lista deve aparecer: "analista de negócios", "analista de dados", "analista de big data", entre outros;

Precisa ser melhor explorado, a lista de opções aparece a partir de quantas letras? Ex colocou an já aparece a opção de analista? E ao clicar em analista aparecem as opções de analista de que?
Tem que descrever mais a fundo esse comportamento

**Comment by Tainawandelli** [comment]
![Image](https://user-images.githubusercontent.com/127143650/229560622-2002c6e5-887a-4dba-8f01-ef370a5ed0ed.png)

**Comment by Tainawandelli** [comment]
No box "estas vagas tem o seu perfil!", o usuário poderá visualizar algumas vagas que correspondam ao seu perfil profissional; Como ocorre esse match? quais os critérios que devem ser usados para saber se o usuário tem o perfil da vaga ou não?

**Comment by samantacds** [comment]
@Jacqueline2021 gostaria de conversar melhor sobre esta US. 
O primeiro ponto é sobre o botão pesquisar vagas, precisamos alinhar com o time de vagas como será exibido o retorno desta consulta
O segundo ponto é sobre o match das vagas, critério 10. Precisamos alinhar com o time de vagas este item.
Critério 11- O que define que o perfil do candidato foi atualizado ou não? Ausência de experiência profissionais? Ausência do cargo?

**Comment by Jacqueline2021** [comment]
Concordo @samantacds precisamos conversar com o time de gestão de vagas. 
No critério 11, eu pensei neste campo como "desatualizado", isto é, conta nova ou muito tempo sem acessar ou atualizar o currículo(ex: 6 meses sem atualização e essa mensagem aparecerá). Preciso colocar estas informações, correto? Aceito sugestões para colocar esse critério ou retirar esse botão. Caso o currículo esteja atualizado dentro do período de 6 meses, as vagas que tenham mais "match" com o perfil do usuário aparecerão neste campo.

**Comment by Tainawandelli** [comment]
Gente viajei desculpa. Me meti onde não devia. Sai validando as us e não me liguei que era de perfil de vagas. Perdão por me intrometer.

**Comment by Jacqueline2021** [comment]
@samantacds para correção: alterações feitas conforme solicitado na reunião. Obrigada.

**Comment by Jacqueline2021** [comment]
@samantacds para correção! Obrigada! 26/04/23
@LuandaClara para criação de tela! Obrigada!

**Comment by samantacds** [comment]
@Jacqueline2021 validada! 28/04/23 US validada.

**Comment by denyscarvalho** [comment]
A taks #164 vinculada a essa US já foi testada e validada pelo time de QA. Aguardando validação do APM @LuizGuilhermeNeves
